github3/repos/branch.py

```diff
@@ -401,6 +543,76 @@ def _update_attributes(self, protection):
             self,
         )
 
+    def set_required_code_owner_reviews(self, value):
```

Comment:
I seem to have not made myself clear before. We do not use objects in this manner in github3.py. Please remove the `set_*` methods here. These should be parameters to `update`.